source/codegen/metadata/nirfmxspecan/functions.py

```diff
@@ -2209,8 +2209,12 @@
                 'type': 'int32'
```

Comment:
Shouldn't selectorStringOutLength be direction out too?

Reply:
No, it's an input to the Driver API method. These ivi-dance ones get the size from the first call where a positive status indicates the size to use.

1. Call API with 0 for `selectorStringOutLength`, returned status (if positive) is the size to use
2. Call API again with status in step one passed in for `selectorStringOutLength`